main/views.py

```diff
@@ -1,5 +1,26 @@
-from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import redirect
+# from dogowner.models import DogOwner
+# from daycare.models import DayCare
+# from daycare.views import daycare_home
+# from dogowner.views import dog_owner_home
 
 
-def homepage(request):
-    return render(request, 'main/homepage.html')
+def index(request):
+    # need to implement home function for logged- in user.
```

Comment:
seems like this PR is not ready for review yet.
please either:
1.  implement, including tests
2.  redirect to a different, logical place, then open an issue to implement this function instead of comments in the code.